generator/transit_generator.cpp

```diff
@@ -189,235 +65,10 @@ string GetMwmPath(string const & mwmDir, TCountryId const & countryId)
-  transit::FixedSizeDeserializer<NonOwningReaderSource> numberDeserializer(src);
-
-  transit::TransitHeader header;
-  numberDeserializer(header);
-  CHECK(header.IsValid(), ());
-
-  CHECK_EQUAL(src.Pos(), header.m_stopsOffset, ("Wrong", TRANSIT_FILE_TAG, "section format."));
-  deserializer(m_stops);
-  CHECK(transit::IsValidSortedUnique(m_stops), ());
-
-  CHECK_EQUAL(src.Pos(), header.m_gatesOffset, ("Wrong", TRANSIT_FILE_TAG, "section format."));
-  deserializer(m_gates);
-  CHECK(transit::IsValidSortedUnique(m_gates), ());
-
-  CHECK_EQUAL(src.Pos(), header.m_edgesOffset, ("Wrong", TRANSIT_FILE_TAG, "section format."));
-  deserializer(m_edges);
-  CHECK(transit::IsValidSortedUnique(m_edges), ());
-
-  CHECK_EQUAL(src.Pos(), header.m_transfersOffset, ("Wrong", TRANSIT_FILE_TAG, "section format."));
-  deserializer(m_transfers);
-  CHECK(transit::IsValidSortedUnique(m_transfers), ());
-
-  CHECK_EQUAL(src.Pos(), header.m_linesOffset, ("Wrong", TRANSIT_FILE_TAG, "section format."));
-  deserializer(m_lines);
-  CHECK(transit::IsValidSortedUnique(m_lines), ());
-
-  CHECK_EQUAL(src.Pos(), header.m_shapesOffset, ("Wrong", TRANSIT_FILE_TAG, "section format."));
-  deserializer(m_shapes);
-  CHECK(transit::IsValidSortedUnique(m_shapes), ());
-
-  CHECK_EQUAL(src.Pos(), header.m_networksOffset, ("Wrong", TRANSIT_FILE_TAG, "section format."));
-  deserializer(m_networks);
-  CHECK(transit::IsValidSortedUnique(m_networks), ());
-
-  CHECK_EQUAL(src.Pos(), header.m_endOffset, ("Wrong", TRANSIT_FILE_TAG, "section format."));
-}
-
-void GraphData::AppendTo(GraphData const & rhs)
-{
-  ::Append(rhs.m_stops, m_stops);
-  ::Append(rhs.m_gates, m_gates);
-  ::Append(rhs.m_edges, m_edges);
-  ::Append(rhs.m_transfers, m_transfers);
-  ::Append(rhs.m_lines, m_lines);
-  ::Append(rhs.m_shapes, m_shapes);
-  ::Append(rhs.m_networks, m_networks);
-}
-
-void GraphData::Clear()
-{
-  ClearVisitor const v{};
-  Visit(v);
-}
-
-bool GraphData::IsValid() const
-{
-  if (!IsSorted() || !IsUnique())
-    return false;
-
-  IsValidVisitor v;
-  Visit(v);
-  return v.IsValid();
-}
-
-bool GraphData::IsEmpty() const
-{
-  // Note. |m_transfers| may be empty if GraphData instance is not empty.
-  return m_stops.empty() || m_gates.empty() || m_edges.empty() || m_lines.empty()
-      || m_shapes.empty() || m_networks.empty();
-}
-
-void GraphData::Sort()
-{
-  SortVisitor const v{};
-  Visit(v);
-}
-
-void GraphData::ClipGraph(vector<m2::RegionD> const & borders)
-{
-  Sort();
-  CHECK(IsValid(), ());
-  ClipLines(borders);
-  ClipStops();
-  ClipNetworks();
-  ClipGates();
-  ClipTransfer();
-  ClipEdges();
-  ClipShapes();
-  CHECK(IsValid(), ());
-}
-
-void GraphData::CalculateBestPedestrianSegments(string const & mwmPath, TCountryId const & countryId)
+/// \brief Calculates best pedestrian segment for every gate in |graphData.m_gates|.
+/// \note All gates in |graphData.m_gates| must have a valid |m_point| field before the call.
```

Comment:
And where the results of these calculations will be put?

Reply:
I'll add to the documentation:
```
/// The result of the calculation is set to |Gate::m_bestPedestrianSegment| of every gate
/// from |graphData.m_gates|.
```